Fix example scripts

cdcapano opened the pull request:
Renames `gwin` -> `pycbc_inference` that was missed in some of the example scripts, and fixes some other debugging things that I forgot to exclude.

soumide1102 commented on examples/inference/analytic-normal2d/make_movie.sh:
```diff
@@ -1,4 +1,4 @@
-gwin_plot_movie --verbose \
+pycbc_inference_plot_movie --verbose \
```
How about adding `loglikelihood` to `--z-arg` and have a label for that in `option_utils.py`?

cdcapano replied:
@soumide1102 Yeah that's right. But in this example, there were only two parameters, so it's ok to not provide a `--parameters` argument.

cdcapano replied:
@soumide1102 Yeah that's right. But in this example, there were only two parameters, so it's ok to not provide a `--parameters` argument.

soumide1102 commented on examples/inference/gw150914/plot.sh:
```diff
@@ -1,35 +1,14 @@
     --plot-marginal \
-    --z-arg 'mchirp_from_mass1_mass2(mass1, mass2):mchirp' \
-    --parameters tc \
-                 'snr_from_loglr(abs(H1_cplx_loglr)):$\rho_H$' \
```
I see that you removed the sample code for plotting the likelihood params colored by the samples params. Isn't this feature supported in the version of the code on inference_dev now? That example will be a nice thing to have in the examples in addition to plotting the samples colored by the likelihood params.
@soumide1102

cdcapano replied:
Yeah, that feature is available but what I had in that script -- plotting the SNR in each detector --- would not work with the GW150914 example. The problem is `emcee_pt` does not support blob data, so at the moment we have no way of getting things like the SNR in each detector. 

The only thing we can (and do) save is the loglikelihood and logprior. I could add an example of plotting one of those on one of the corner axes, but I'm not sure how useful that will be. That sort of example is probably best described in the [plotting the posterior pages](https://www.atlas.aei.uni-hannover.de/~cdcapano/public_scratch/github_prs/inference_dev-test_docs_update/html/inference/viz.html), which I haven't touched yet. Some more elaborate examples should be included in that page, but that should be done in separate PR.

As for the detector-specific statistics: only samplers that support blob data, like `emcee`, allow us to save the extra statistics. (It was in this script because this was a test I had done with `emcee`.) Unfortunately, `ptemcee` doesn't currently support blob data at the moment either. I opened an [Issue to ask them to add it](https://github.com/willvousden/ptemcee/issues/7), but we may need to file a pull request ourselves on `ptemcee` to get it to be added. And I don't know how this will work in cpnest and multinest.

Anyway, once we get support for `ptemcee` and get `ptemcee` to handle blob data, then we can provide an example of plotting the SNR in each detector.

cdcapano commented:
@soumide1102 Just a heads up, I also addressed some issues [Codacy complained about](https://app.codacy.com/app/thomas-dent/pycbc/pullRequest?prid=2396915) in PR #2389. I added shebangs to the top of the sh files, and removed some unnecessary `${}` in the BBH and GW150914 `run.sh` files.
